src/core/qgstemporalproperty.h

```diff
@@ -58,8 +59,16 @@ class CORE_EXPORT QgsTemporalProperty
     */
     bool isActive() const;
 
+  signals:
+
+    /**
+     * Emitted when the temporal properties have changed.
+     */
+    void changed();
```

Comment:
Can you add unit tests checking that this is correctly emitted when the various settings change (and not re-emitted when the existing value is re-set)